lib/rubygems/commands/fetch_command.rb

```diff
@@ -18,6 +18,10 @@ def initialize
     add_version_option
     add_platform_option
     add_prerelease_option
+
+    add_option '--[no-]suggestions', 'Suggest alternates when gems are not found' do |value, options|
+      options[:suggest_alternate] = true
```

Comment:
I think this should be assigned to `value`, and pass a default `true` value when calling `super` above.